python/ray/train/torch.py

```diff
@@ -132,6 +137,25 @@ def with_sampler(loader):
                 # SequentialSampler.
                 shuffle = not isinstance(loader.sampler, SequentialSampler)
 
+                def seed_worker(worker_id):
+                    worker_seed = torch.initial_seed() % 2 ** 32
```

Comment:
oh this is interesting, out of curiosity, why do the % 2 **32?

Reply:
I think it's because the `np.random.seed` requires a seed that is "convertible to 32 bit unsigned integers."